ui/js/models/perf/series.js

```diff
@@ -63,11 +63,11 @@ treeherder.factory('PhSeries', ['$http', '$q', function ($http, $q) {
                 return $http.get(
                     getProjectUrl('/performance/signatures/', projectName),
                     { params: params }).then(function (response) {
-                         return _.map(response.data, function (signatureProps, signature) {
```

Comment:
this is another case where ``response.data`` is not an array, but an object.  So we would need to use ``Object.keys`` with some destructuring:

    return Object.entries(response.data).map(([signature, signatureProps]) => (
        ...
    )